mani/utils.py

```diff
@@ -1,2 +1,82 @@
-"""Add your final methods here."""
+import random
+import webbrowser
+import time
+
+# Defining the items
+
+fruits = ["apple", "banana", "mango"]
+vegs = ['carrot', 'beans', 'potato']
+drinks = ['milk', 'water', 'juice']
+
+items = fruits+vegs+drinks
+
+fruit_url="https://en.wikipedia.org/wiki/Fruit"
+
+drink_url="https://en.wikipedia.org/wiki/Drink"
+
+veg_url="https://en.wikipedia.org/wiki/Vegetable"
+
+def what_is_the_item(category):
+      if category in fruits:
+            print("\n", category + " is a fruit")
+            time.sleep(3)
```

Comment:
not sure why time.sleep used here ?

any reason

Reply:
To give the user some time to see the output and then it navigated to wiki page to give brief about the item category